.travis.yml

```diff
@@ -2,4 +2,30 @@ language: java
 sudo: false
```

Comment:
@erikdw yep, you are right, the releases are created on tags. I' assuming you only create tags on new releases.